src/common/components/Input/index.tsx

```diff
@@ -138,15 +187,21 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(({ label, error, w
                         </InputLabel>
                     )}
 
-                    <InputElement
-                        id={id}
-                        hasError={indicateError}
-                        aria-invalid={indicateError}
-                        showAlertMessage={showErrorMessage || showWarningMessage}
-                        ref={ref}
-                        {...rest}
-                        onBlur={makeDirty}
-                    />
+                    <Field inputWidth={inputWidth} >
+                        {hide && <IconSpan show={show} onClick={show ? () => setShow(false) : () => setShow(true)}>{show ? viewOn : viewOff}</IconSpan>}
```

Comment:
Could simplify this:

```suggestion
                        {hide && <IconSpan show={show} onClick={setShow(!show)}>{show ? viewOn : viewOff}</IconSpan>}
```